lab_3_ann_retriever/main.py

```diff
@@ -194,6 +284,14 @@ def save(self, file_path: str) -> bool:
         Returns:
             bool: True if saved successfully, False in other case
         """
+        state = {'idf_values': self._idf_values,
+                 'vocabulary': self._vocabulary,
+                 'token2ind': self._token2ind}
+        with open(file_path, 'w', encoding='utf-8') as file:
+            dump(state, file)
+        if os.stat(file_path).st_size == 0:
```

Comment:
Do you really need this?

Reply:
otherwise how can I make a check for successful state saving?

Reply:
just skip - remove this strange check

Reply:
done